src/Sulu/Bundle/PageBundle/Document/BasePageDocument.php

```diff
@@ -509,4 +516,14 @@ public function setAuthor($author)
     {
         $this->author = $author;
     }
+
+    public function setSegment(?string $segment = null)
```

Comment:
Neither these functions nor a separate behavior would be needed when moved to excerpt tab.